.gitignore

```
@@ -98,3 +98,6 @@ ENV/
 work/
 testdata/
 .vscode/
+
+nibabies/_version.py
+wrapper/**/_version.py
```

Simpler and less brittle if you reorganize:

```suggestion
_version.py
```